shaders/mesh_shader_culling/mesh_shader_shared.h

```diff
@@ -0,0 +1,28 @@
+/* Copyright (c) 2023, Holochip Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 the "License";
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Example of the data shared with its associated mesh shader:
+// 1) define some structure if more than one variable data sharing is desired:
+
+struct SharedData
+{
+	vec4  positionTransformation;
+	int   N;
+	int   meshletsNumber;
+	float subDimension;
+	float cullRadius;
+};
```

Comment:
In order to reduce payload size: `N` and `meshletsNumber` are not used in the mesh shader, `subDimension` could be easily calculated out of a constant and `gl_WorkGroupSize`. And from `positionTransformation`, only `x` and `y` need to be carried. Thus the size of `SharedData` could be reduced from 8 * 4 byte to 3 * 4 byte.
And one could even argue, that those three values are part of an UBO, that could be passed into the mesh shader as well, instead of the payload. But in order demonstrate how a payload is managed, it should probably be kept, maybe with a comment on how to remove it.